### DISCUSSION TITLE
yarn build error

### DISCUSSION THREAD

**Discussion by aniwn88480857** [umami-software/umami#1282] [Q&A] [answered]
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: Get config: Unable to establish a connection to query-engine-node-api library

Prisma CLI Version : 3.15.2
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-db-client" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "build-db" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by mikecao** [comment, marked as answer]
You may need to install some additional packages for your OS. See https://github.com/prisma/prisma/issues/13717.
